librz/core/cmd_descs/cmd_print.yaml

```diff
@@ -1447,6 +1447,25 @@ commands:
           - name: skip
             type: RZ_CMD_ARG_TYPE_RZNUM
             optional: true
+      - name: p=r
+        summary: Print rising and falling entropy by default rising threshold is 0.95 and falling threshold is 0.85
+        cname: print_rising_and_falling_entropy
+        args:
```

Comment:
Please it make `RzCmdStateOutput` and add these modes:
- JSON
- QUIET - just a list of addresses
- SHORT - the address plus if raising or falling edge
- LONG - current output
- TABLE - same but as table